invenio_theme/assets/semantic-ui/js/invenio_theme/Media.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2017-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import { createMedia } from "@artsy/fresnel";
```

Comment:
What's the purpose of this addition? Sorry, I miss some context.....

Reply:
this is the library which is recommended by semantic to handle media components (the ones responsible for differentating the looks of react components for different devices)